tests/labhub_test.py

```diff
@@ -241,6 +256,8 @@ def test_assign_cmd(self):
         mock_maint_team = create_autospec(github3.orgs.Team)
         mock_dev_team.is_member.return_value = False
         mock_maint_team.is_member.return_value = False
+        mock_nick = create_autospec(Message)
```

Comment:
please undo all of your changes to other test methods.

First get one new test method to work, then you can duplicate it.

Reply:
Doing this right now.